packages/documentation/src/pages/getting-started/common-tasks/new-end-to-end-test.mdx

```diff
@@ -11,7 +11,9 @@ Front end engineers use end-to-end (e2e) tests in `vets-website` to validate mul
 
 ## End-to-end testing overview
 
-- `vets-website` uses [Nightwatch](https://nightwatchjs.org) to run the tests and provide the browser client
+- `vets-website` uses [Nightwatch](https://nightwatchjs.org) to run some of the older end-to-end tests
+  - New end-to-end tests should be written in Cypress going forward.
+  - Refer to the [Cypress migration guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/6dccdd3ac61d7fbfa8b30e317230cc9212a7c9f8/teams/vsp/teams/tools/frontend/cypress-migration-guide.md) to convert old tests or follow certain conventions with new tests.
```

Comment:
```suggestion
  - Refer to the [Cypress migration guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsp/teams/tools/frontend/cypress-migration-guide.md) to convert old tests or follow certain conventions with new tests.
```

Reply:
link to latest version of docs

Reply:
I prefer linking to a specific commit's version, since that's much less likely to result in a broken link. Documents have a tendency to move around, and I anticipate this might, so I'm not keen on using a "current" link.